### PR TITLE
Documentation fixes.

### PR DESCRIPTION
**Pull request by nawroth**
- Removes references to test utils from batchinserter examples.
- Javadoc cleanup in BatchInserters.java.
- Better updating queries page.
- Fix for rendering of config attribute options.
- Make the scope of PERIODIC COMMIT clear.
- Correct max for relationship types.
- Add example creating multiple nodes.
- Add example sending multiple statements to tx endpoint.
- Add example for getting query stats from the tx endpoint.
- Add regex operator to the operators page.

### CODE REVIEW

File: community/cypher/docs/cypher-docs/src/docs/dev/general/updating.asciidoc [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
 
 Cypher can be used for both querying and updating your graph.
 
 [[query-updating-structure]]
 == The Structure of Updating Queries ==
 
-.Quick info
-***********
+[abstract]
 * A Cypher query part can't both match and update the graph at the same time.
```
Comment (Mats-SX) [inline community/cypher/docs/cypher-docs/src/docs/dev/general/updating.asciidoc]: That's not true, right?

File: community/cypher/docs/cypher-docs/src/docs/dev/general/updating.asciidoc [hunk 2]
```diff
@@ -32,20 +29,31 @@ WHERE friendsCount > 3
...
 
-Using `WITH`, you specify how you want the aggregation to happen, and that the aggregation has to be finished before
-Cypher can start filtering.
+Using `WITH`, you specify how you want the aggregation to happen, and that the aggregation has to be finished before Cypher can start filtering.
+
+Here's an example of updating the graph, writing the aggregated data to the graph:
+
+[source,cypher]
+----
+MATCH (n {name: 'John'})-[:FRIEND]-friend
```
Comment (Mats-SX) [inline community/cypher/docs/cypher-docs/src/docs/dev/general/updating.asciidoc]: I would prefer wrapping all matched nodes in parenthesis, ie `(friend)` here.

File: community/cypher/docs/cypher-docs/src/docs/dev/general/updating.asciidoc [hunk 3]
```diff
@@ -32,20 +29,31 @@ WHERE friendsCount > 3
...
-After all the parts of the query comes one final `RETURN` clause. `RETURN` is not part of any query part -- it
-is a period symbol at the end of a query. The `RETURN` clause has three sub-clauses that come with it `SKIP`/`LIMIT` and `ORDER BY`.
+After all the parts of the query comes one final `RETURN` clause.
+`RETURN` is not part of any query part -- it is a period symbol at the end of a query.
+The `RETURN` clause has three sub-clauses that come with it: `SKIP`/`LIMIT` and `ORDER BY`.
 
-If you return graph elements from a query that has just deleted them -- beware, you are holding a pointer that is no
- longer valid. Operations on that node might fail mysteriously and unpredictably.
+If you return graph elements from a query that has just deleted them -- beware, you are holding a pointer that is no longer valid.
+Operations on that node might fail.
```
Comment (Mats-SX) [inline community/cypher/docs/cypher-docs/src/docs/dev/general/updating.asciidoc]: What about `Operations on that node are undefined`?

File: community/cypher/docs/cypher-docs/src/docs/dev/syntax/operators.asciidoc [hunk 4]
```diff
@@ -21,6 +21,7 @@ The boolean operators are `AND`, `OR`, `XOR`, `NOT`.
 == String operators ==
 
 Strings can be concatenated using the `+` operator.
+For regular expressions, the `=~` operator is used.
```
Comment (Mats-SX) [inline community/cypher/docs/cypher-docs/src/docs/dev/syntax/operators.asciidoc]: What about `For regular expression matching,[...]`?

File: community/cypher/docs/cypher-docs/src/docs/dev/syntax/values.asciidoc [hunk 5]
```diff
@@ -2,7 +2,7 @@
 Values
 ======
 
-Cypher queries the graph by looking at nodes, relationships, properties and query parameters.
+Cypher queries the graph by looking at nodes, relationships, properties, labels and query parameters.
```
Comment (Mats-SX) [inline community/cypher/docs/cypher-docs/src/docs/dev/syntax/values.asciidoc]: Are literals considered query parameters in this case?

Comment (nawroth) [inline community/cypher/docs/cypher-docs/src/docs/dev/syntax/values.asciidoc]: How about we just remove that sentence, I'm not sure it's helpful anyhow?

Comment (Mats-SX) [inline community/cypher/docs/cypher-docs/src/docs/dev/syntax/values.asciidoc]: Sure.

**Pull request merged**